lib/views/pay/pay_view.dart

```diff
@@ -1,102 +1,214 @@
         children: [
           TextField(
+            minLines: 5,
+            maxLines: 10,
+            controller: _invoiceController,
             onChanged: (invoiceBolt) {
               boltString = invoiceBolt;
             },
-            decoration: const InputDecoration(
-              border: OutlineInputBorder(),
+            decoration: InputDecoration(
+              enabledBorder: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(20),
+                borderSide: const BorderSide(
+                  color: Color.fromARGB(255, 255, 121, 197),
+                  width: 1.0,
+                ),
+              ),
+              focusedBorder: OutlineInputBorder(
+                borderRadius: BorderRadius.circular(20),
+                borderSide: const BorderSide(
+                  color: Color.fromARGB(255, 255, 121, 197),
+                  width: 1.0,
+                ),
+              ),
+              alignLabelWithHint: true,
+              contentPadding: const EdgeInsets.all(20),
               labelText: 'Invoice Bolt11/12',
               hintText: 'Bolt11/12',
             ),
           ),
-          Row(
-            children: [
-              Checkbox(
-                value: amountMsat == null ? false : true,
-                onChanged: (value) {
-                  setState(() {
-                    amountMsat = (value == true ? 0 : null);
-                  });
-                },
-              ),
-              const Text(
-                'Enter milli-satoshi amount?',
-                style: TextStyle(fontSize: 17.0),
-              ), //Text//SizedBox
-            ],
-          ),
-          TextField(
-            keyboardType: TextInputType.number,
-            inputFormatters: <TextInputFormatter>[
-              FilteringTextInputFormatter.allow(RegExp(r'[0-9]')),
-            ],
-            onChanged: (amount) {
-              amountMsat = int.parse(amount);
-            },
-            enabled: amountMsat == null ? false : true,
-            decoration: const InputDecoration(
-              border: OutlineInputBorder(),
-              disabledBorder: OutlineInputBorder(),
-              labelText: 'Amount to send in millisatoshi',
-              hintText: 'Eg: 100 msats',
-            ),
-          ),
           MainCircleButton(
-              icon: Icons.send_outlined,
-              label: "Pay",
-              onPress: () {
-                if (boltString == null || boltString!.isEmpty) {
-                  setState(() {
-                    error = "Error: Bolt11/12 Invoice required";
-                  });
-                } else {
-                  payInvoice(boltString!, amountMsat).then((value) => {
-                        setState(() {
-                          paymentResponse = value;
-                          error = null;
-                        }),
-                      });
+              icon: Icons.copy,
+              label: "Copy from clipboard",
```

Comment:
```suggestion
              label: "Paste from clipboard",
```

Better paste here? :)